docs/authoring.md

```diff
@@ -1,49 +1,51 @@
 
 ## References
 Here are some resources that demonstrate authoring C#/WinRT components and the details discussed in this document.
-1. https://github.com/microsoft/CsWinRT/tree/master/src/Tests/AuthoringTest
+1. [Simple C#/WinRT component sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/AuthoringDemo) and associated [walkthrough](https://docs.microsoft.com/en-us/windows/uwp/csharp-winrt/create-windows-runtime-component-cswinrt) on creating a C#/WinRT component and consuming it from C++/WinRT
 
-2. https://github.com/microsoft/CsWinRT/tree/master/src/Tests/AuthoringConsumptionTest
-
-3. https://github.com/AdamBraden/MyRandom
+2. https://github.com/microsoft/CsWinRT/tree/master/src/Tests/AuthoringTest
 
+3. https://github.com/microsoft/CsWinRT/tree/master/src/Tests/AuthoringConsumptionTest
 
 ## Authoring the C# Component
 To create a library, select the Class Library (.NET Core) template in Visual Studio. C#/WinRT projects require Windows API version specific .NET frameworks.
 
-Accepted `<TargetFramework>` properties |
+Accepted `<TargetFramework>` versions |
 --- |
 `net5.0-windows10.0.19041.0` |
 `net5.0-windows10.0.18362.0` |
 `net5.0-windows10.0.17763.0` |
 
 The library you are authoring should specify the following properties in its project file: 
+
 ``` csproj
 <PropertyGroup>
   <!-- update the Windows API version to reflect your TargetFramework -->
-  <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
+  <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
   <CsWinRTComponent>true</CsWinRTComponent>
+  <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
   <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
-  <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-  <GeneratedFilesDir Condition="'$(GeneratedFilesDir)'==''">$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(IntermediateOutputPath)', 'Generated Files'))</GeneratedFilesDir>
 </PropertyGroup>
 ```
 And don't forget to include a `PackageReference` to `Microsoft.Windows.CsWinRT`!
 
+## Generate a NuGet package for the component
+You can add the following property to the library project file to automatically generate a NuGet package.
 
-## Using your authored component
-To use the component in a C# app, the authored component just needs to be added as a project/package reference.
-
-For native (C++) apps, there are DLLs needed to host your authored component. When you use the automatic nuget packaging on-build support (in Visual Studio) to make a nupkg for your runtime component, the DLLs/WinMD are automatically added to your nupkg, before the ```GenerateNuspec``` MSBuild step.
+``` csproj
+<PropertyGroup>
+  <GeneratePackageOnBuild>true</GeneratePackageOnBuild> 

```

Comment:
Also, this is kind of misleading. You don't have to specify this for our targets to go into work. You could just right click the project and click "Pack" and our packaging logic kicks in.